openfisca_us/parameters/benefit/hhs/smi_one_person.yaml

```diff
@@ -0,0 +1,12 @@
+description: State median income for 1-person households by state
```

Comment:
This is unnecessary - HHS applies a constant adjustment factor across all states: https://github.com/PolicyEngine/openfisca-us/issues/263

e.g. this is 52% of the 4-person. So we only need the 4-person.